src/ansys/fluent/core/__init__.py

```diff
@@ -44,16 +44,82 @@ def version_info() -> str:
     return _VERSION_INFO if _VERSION_INFO is not None else __version__
 
 
+def set_global_log_level(level: str | int) -> None:
```

Comment:
I think `|` in type-hint will not work in python 3.8

Reply:
You are right, going to change it to a Union[] which works in Python 3.8 according to my tests. Thank you for the heads-up!